sdk/appconfiguration/app-configuration/test/public/tracing.spec.ts

```diff
@@ -1,26 +1,34 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { Recorder } from "@azure-tools/test-recorder";
+import { assert, instrumenter } from "@azure-tools/test-utils";
 import { createAppConfigurationClientForTests, startRecorder } from "./utils/testHelpers";
+
 import { AppConfigurationClient } from "../../src/appConfigurationClient";
 import { Context } from "mocha";
-import { assert } from "@azure-tools/test-utils";
+import { Recorder } from "@azure-tools/test-recorder";
 
-describe("supports tracing", () => {
+describe.only("supports tracing", () => {
   let client: AppConfigurationClient;
+  let fixedClient: AppConfigurationClient;
   let recorder: Recorder;
 
   beforeEach(async function (this: Context) {
     recorder = await startRecorder(this);
     client = createAppConfigurationClientForTests(recorder.configureClientOptions({}));
+    fixedClient = createAppConfigurationClientForTests(
+      recorder.configureClientOptions({
+        instrumenter,
+      }),
+    );
   });
 
   afterEach(async () => {
     await recorder.stop();
   });
 
   it("can trace through the various options", async function () {
+    console.log("expected to fail with missing spans");
```

Comment:
To avoid dealing with recorder, I just have two tests with the exact same name, but the console log should match and we should see one pass and one fail